constructor/sh/construct.yaml

```diff
@@ -0,0 +1,28 @@
+name: omnisci
+version: 0.0.2
```

Comment:
Does this number need to be updated automatically somehow?

Reply:
I think so. My understanding is that `rever´ (https://github.com/Quansight/omnisci-datascience-installer/pull/12/files#diff-fd1836674becaae59f74476b73a08dd3R10-R11) does that. Is that correct?

I didn't update the version yet because I didn't change the pkg recipe. But it is ready for a new release when a new tag is received. I can cut a patch release in a follow-up PR to remove `conda` dependency for `sh`.